app/game/game.js

```diff
@@ -1,27 +1,61 @@
 const { ROLES, GAME_TICK_DELAY_MS } = require("../constants");
-const { Threat, THREAT_COOLDOWN_SECONDS } = require("./threat");
-const { randomInt } = require("../utils.js");
+const { Threat, THREAT_COOLDOWN_SECONDS, MAX_THREATS_ACTIVE } = require("./threat");
+const { randomInt, randomChoice } = require("../utils.js");
+const { RandomBag } = require("../randomBag.js");
+
+const { ConsoleLinesLog } = require("../consoleLinesLog.js");
+const { ConsoleLineData } = require("../dataObjects.js");
+const { Room } = require("./room.js");
 
 const GAMES = {
     
 };
 
-const selectThreatRoom = (avilableRooms, roomsWithThreats) => {
-    const rooms = avilableRooms.filter((room) => !(room in roomsWithThreats));
-    return rooms[randomInt(0, rooms.length - 1)];
+const selectThreatRoom = (availableRooms, roomsWithThreats) => {
+    const roomCandidates = availableRooms.filter((room) => !(room in roomsWithThreats));
+    return roomCandidates[randomInt(0, roomCandidates.length - 1)];
 }
 
 const GAME = (humanUsername, aiUsername) => {
-    let gameTime = 4*60;
+    const gameTime = 4*60;
```

Comment:
If we're using timeRemaining as the decrementing time, we don't need gameTime here. It just gets assigned to timeRemaining.

Reply:
This makes sense. We may eventually need to track both the remaining time and the total time, like if we had a big long progress bar at the top of the screen representing the time until rescue. But I'll consolidate this for now